### PR TITLE
:lipstick:

### PR DESCRIPTION
**Pull request by SamVerschueren**
Did some minor tweaks.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-extend-native */
 Array.prototype.findIndex = undefined;
 import test from 'ava';
-import fn from './';
+import fn from '.';
```
Comment (sindresorhus) [inline test.js]: I'm keeping this for consistency. https://github.com/sindresorhus/eslint-plugin-xo/issues/9

Comment (sindresorhus) [inline test.js]: You could `fn` => `m` though

**Comment by SamVerschueren** [comment]
fixed

**Comment by SamVerschueren** [comment]
done :)

**Comment by sindresorhus** [comment]
🍻

**Comment by SamVerschueren** [comment]
Nice 🍻 !

**Pull request merged**